src/ressa/context.rs

```diff
@@ -10,31 +10,34 @@ const RESOLVES_TO: &'static str = "???";
 const TRANSIENT: &'static str = "";
 
 /// Context used by the Parser, storing local variables (#{varname}) and objects/tags
-#[derive(Default, Debug, Clone, Any, PartialEq)]
```

Comment:
Do we no longer need context comparisons?

Reply:
It can't derive it for Object the new object type, so I had to remove it.